driver-core/src/main/com/mongodb/client/model/Aggregates.java

```diff
@@ -683,17 +683,16 @@ public static Bson search(final SearchOperator operator) {
      *
      * @param operator A search operator.
      * @param options Optional {@code $search} pipeline stage fields.
-     * Specifying {@code null} is equivalent to either specifying {@link SearchOptions#defaultSearchOptions()}
-     * or calling {@link #search(SearchOperator)}.
+     * Specifying {@link SearchOptions#searchOptions()} is equivalent to calling {@link #search(SearchOperator)}.
      * @return The {@code $search} pipeline stage.
      *
      * @mongodb.atlas.manual atlas-search/query-syntax/#-search $search
      * @mongodb.atlas.manual atlas-search/operators-and-collectors/#operators Search operators
      * @mongodb.atlas.manual atlas-search/scoring/ Scoring
      * @since 4.7
      */
-    public static Bson search(final SearchOperator operator, @Nullable final SearchOptions options) {
-        return new SearchStage("$search", notNull("operator", operator), options);
+    public static Bson search(final SearchOperator operator, final SearchOptions options) {
```

Comment:
Doesn't `search(final SearchOperator operator)` need updating?

Reply:
Of course 🤦‍♂️. Fortunately, tests failed because of this